mkdocs/mkdocs.yml

```diff
@@ -53,9 +53,20 @@ plugins:
   - search
   - minify:
       minify_html: true
+  - macros:
+      # Replace {{ variable }} syntax with [[ variable ]] to avoid conflicts
+      j2_block_start_string: '[[%'
+      j2_block_end_string: '%]]'
+      j2_variable_start_string: '[['
+      j2_variable_end_string: ']]'
         
 # Customization
 extra:
+  # Google Analytics
```

Comment:
@kalantar This is the new method of adding Google Analytics. You can read more about it [here](https://github.com/iter8-tools/iter8/pull/981#discussion_r693106545).